modules/questions.py

```diff
@@ -415,15 +415,16 @@ async def autopost_not_started(self) -> None:
     def is_time_for_autopost_wip(self) -> bool:
         now = datetime.now()
         return (
-            now.weekday() in (0, 2, 4)  # Monday, Wednesday, or Friday
+            now.weekday() in (0, 4)  # Monday or Friday
```

Comment:
(for future) I would prefer to have some documentation about the reasons for changes like this ... presumably discussed on some meeting but if there is no link to meeting notes or Discord message in this PR and/or linked GH issue, no one will remember why this line of code changed today 🤷 

not a big deal for this particular change, but in 5 years we might wish we had a better habit of documenting who+why requested some of the bigger changes 😅

Reply:
Valid, I added the description to the PR, thanks!